apps/src/templates/manageStudents/ControlProjectSharingDialog.jsx

```diff
@@ -45,10 +43,7 @@ class ControlProjectSharingDialog extends Component {
         >
           <h2>{i18n.projectSharingDialogHeader()}</h2>
           <div>
-            {i18n.projectSharingDialogInstructions1()}
-          </div>
-          <div style={styles.instructions}>
```

Comment:
Can I assume that whoever owns this component has signed off on this styling change?

Reply:
I think that's me? LGTM, but @nkiruka can you do a before and after shot to confirm, please?

Reply:
@Erin007 I have updated PR with before and after screen shots.